install.sh

```diff
@@ -456,7 +456,7 @@ nvm_do_install() {
 
   nvm_reset
 
-  nvm_echo "=> Close and reopen your terminal to start using nvm or run the following to use it now:"
+  nvm_echo "=> Close and reopen your terminal to start using nvm or run the command \`source ${NVM_PROFILE}\` or run the following to use it now:"
```

Comment:
it's far better to have the user close and reopen the terminal, because that way they can test the lines in their profile file.

Reply:
I do not think so. The changes just now only provide more choices, and it is still up to the user to decide which solution to use.

Reply:
I don't want the user to decide - I want the user to close and reopen their terminal, so they can test the lines in their profile file.

Reply:
```sh
=> Appending nvm source string to /home/eric/.bashrc
=> Appending bash_completion source string to /home/eric/.bashrc
```

Moreover, the configuration command has been injected into the configuration file, why do I need to add it manually? That makes no sense.

Reply:
You shouldn’t need to add it manually, the install script adds it for you.